wagtail/tests/modeladmintest/wagtail_hooks.py

```diff
@@ -91,6 +92,41 @@ class VenuePageAdmin(ModelAdmin):
     exclude_from_explorer = True
 
 
+class PersonAdmin(ModelAdmin):
+    model = Person
+    exclude_from_explorer = True
```

Comment:
The `exclude_from_explorer` option is only relevant when `model` is a subclass of `Page`. Since these new models are just standard Django models, the option should be removed for clarity.

Reply:
@ababic fixed in https://github.com/wagtail/wagtail/pull/4900/commits/61d2e4dbdd7049a9e31150f8782d74c344a60c7a